include/dr/algorithms/algorithms.hpp

```diff
@@ -199,7 +193,24 @@ void copy(int root, std::nullptr_t, std::size_t size,
                 scatter_data_dst_size(result, size, comm.rank()), root);
 }
 
-///// Collective copy from local range to distributed iterator
+/// Collective copy from local begin/end to distributed, which is called on
+/// the non-source rank without providing the size of the source (if I is the
+/// same as std::nullptr_t). Collective copy from local begin/end to
```

Comment:
```suggestion
/// same as `std::nullptr_t`). Collective copy from local begin/end to
```
Delimiters excludes it from spell check and will also format it appropriately.